hilo.rb

```diff
@@ -1,26 +1,28 @@
+# Devtrain: Ruby Hi-Lo
+
 # predefined number of guesses allowed
 guesses = 6
 ## computer secretly picks number between 1 and 100 inclusive
 number = rand(1...101)
 
 # conditional logic that is run as long as user has remaining guesses left
 while guesses > 0
-    # get user input
-    puts "Guess the number between 1 & 100 (inclusive):"
-    # store user input
-    guess = gets.chomp.to_i
-    # if user guesses correctly, game is won (guess = number)
-    if (guess == number)
-        puts "You win! #{number} was the number!"
-        break
-    # if guess too high, decrement number of guesses left and tell them their guess was too high
-    elsif guess > number
-        guesses -= 1
-        puts "Your guess is too high!"
-    # if guess too low, decrement number of guesses left and tell them their guess was too low
-    elsif guess < number
-        guesses -=1
-        puts "Your guess is too Low!"
+  # get user input
+  puts "Guess the number between 1 & 100 (inclusive):"
+  # store user input
+  guess = gets.chomp.to_i
+  # if user guesses correctly, game is won (guess = number)
+  if (guess == number)
+    puts "You win! #{number} was the number!"
+    break
+  # if guess too high, decrement number of guesses left and tell them their guess was too high
+  elsif guess > number
+    guesses -= 1
+    puts "Your guess is too high!"
+  # if guess too low, decrement number of guesses left and tell them their guess was too low
+  elsif guess < number
+    guesses -=1
```

Comment:
Now I'm just being picky 😂 but you have `guesses -= 1` when you could have it outside the if statement. It's also a bit more future proof because if you refactor these if statements you won't have to worry about the incrementing.

Reply:
done!